src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_ID = 0;
+    private static final int AMOUNT_ID = 1;
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String SEPARATOR = ",";
+    private static final String RESULT = "result";
```

Comment:
don't need to make constants from these values and create variables for them too